codalab/model/worker_model.py

```diff
@@ -188,13 +190,36 @@ def get_workers(self):
                 and self._deserialize_dependencies(row.dependencies),
                 'shared_file_system': row.shared_file_system,
                 'tag_exclusive': row.tag_exclusive,
+                'exit_after_num_runs': row.exit_after_num_runs,
             }
             for row in worker_rows
         }
         for row in worker_run_rows:
             worker_dict[(row.user_id, row.worker_id)]['run_uuids'].append(row.run_uuid)
         return list(worker_dict.values())
 
+    def update_workers(self, user_id, worker_id, update):
+        """
+        Returns information about all the workers in the database. The return
```

Comment:
This doesn't just return information, but also updates the table if an update is provided, right? Might be worth updating the docstring in this case.

Reply:
Yup. Updated!